packages/proxy-container/tsup.config.ts

```diff
@@ -0,0 +1,19 @@
+import { defineConfig } from 'tsup';
+
+export default defineConfig({
+	clean: true,
+	entryPoints: ['src/index.ts'],
+	format: ['esm', 'cjs'],
```

Comment:
Do we even need a multi-format bundle? For what's worth, this is an app and not a library.

Reply:
True, we could just bundle as `esm` only 👍

Reply:
Should we also use minify and other options to keep bundle size the smallest?

Reply:
Yes sure, we could.

Reply:
Anything else other than `minify`?

Reply:
Well, you could update to use the root `tsup.config.ts` config, since it has defaults already.

Reply:
`skipNodeModulesBundle: false`, `minify: true`, and I'm unsure if we need source maps, probably?

Reply:
`skipNodeModulesBundle: false` that doesn't exactly bundle `node_modules`, you'd have to use another option which I have to look up.